src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -40,7 +40,11 @@
             <DataSource Persistence="ProjectFileWithInterception" PersistedName="TargetFrameworkMoniker" HasConfigurationCondition="False" />
         </StringProperty.DataSource>
     </StringProperty>
-
+    <StringProperty Name="TargetFrameworks" DisplayName="List of Supported Target Frameworks">
```

Comment:
You're returning "nuget" friendly names - but they should be Target Framework Monikers, which will be statically created based on this property.